modules/vector-search/pages/create-vector-search-index-ui.adoc

```diff
@@ -53,12 +53,15 @@ You cannot have 2 indexes with the same name inside the same bucket and scope.
 . Expand *Customize Index*. 
 . Select *Use non-default scope/collection(s)*.
 .. In the *Scope* list, select the scope that contains the documents you want to include in your index.
-. Expand *Mappings* and xref:search:create-type-mapping.adoc[create a new type mapping on a collection]:
+. Expand *Mappings* and create a new type mapping on a collection:
 .. Click btn:[Add Type Mapping].
 .. In the *Collection* list, select the collection that contains the documents you want to include in your index. 
 .. Select *Only index specified fields*. 
 .. Click btn:[OK].
-. xref:search:create-child-field.adoc[] on the new collection type mapping with the following settings: 
+
++
+For more information about how to create type mappings, see xref:search:create-type-mapping.adoc[].
+. Create a child field mapping on the new collection type mapping: 
 .. In the *Field* field, enter the name of the field in your documents that contains your vector embeddings.
```

Comment:
```suggestion
.. In *Field*, enter the name of the field in your documents that contains your vector embeddings.
```

Reply:
The original sentence is fine. I included this in the review because I wanted to try another option to reduce the number of "fields", but again, just a suggestion!

Reply:
Yeah I considered that, but then it loses parity with the rest of the sentences and feels strange to me.

Reply:
No problem, I'll go approve!